sources/platform/console/authentication.md

```diff
@@ -0,0 +1,136 @@
+
+In this view you can use your favorite authenticator app to scan the QR code. We recommend using Google Authenticator ([Google play store](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US)/[Apple app store](https://apps.apple.com/us/app/google-authenticator/id388497605)) or [Authy](https://authy.com/) or but any other authenticator app should work as well.
+
+You can also setup your app/browser extension manually without the QR code. To do that, click on the **Setup key** link below the QR code this view with key will pop up:
+
+![Apify console setup two-factor authentication - key](./images/console-setup-two-factor-auth-key.png)
+
+Here you can copy the two-factor `secret` key, which you can use in your authenticator app to setup the account. You can also use this key to setup your authenticator app on multiple devices.
+
+After you scan the QR code, or setup your app manually, the app will generate a code which you need to enter into the **Verify the code from the app** field. After you enter the code, click on the **Continue** button to get to the next step of the setup process.
+
+![Apify console setup two-factor authentication - recovery codes](./images/console-setup-two-factor-auth-codes.png)
+
+In this step you will we will show you 16 recovery codes. These codes can are required when you want to sign in to your account if you lose access to your authenticator app. We recommend saving these codes in a safe place, ideally printing them out and not storing them on your device.
+
+You will not be able to enable the two-factor authentication until you on the **Download** / **Copy** buttons, or copy the codes manually. After you do that, the **I have saved my recovery codes** button will be enabled and you can click on it to enable the two-factor authentication. Which will be confirmed in the next view.
+
+### Verification after sign in
+
+After you enable two-factor authentication, the next time you attempt to sign in, you will not be directly redirected to Apify Console, but instead you will have to provide your verification code. To do that, open your authenticator app and enter the code for your Apify account into the **code** field. After you enter the code, click on the **Verify** button and if the provided code is correct, you will proceed to Apify Console.
+
+![Apify console two-factor authentication form](./images/console-two-factor-authentication.png)
+
+### Using recovery codes
+
+In case you loose access to your authenticator app, you can use the recovery codes to sign in to your account. To do that, click on the **recovery code or begin 2FA account recovery** link below the **Verify** button. This will redirect you to a similar view as the current one, but instead of code from the authenticator app, you will need to enter one of your 16 recovery codes which you have received during the setup process.
```

Comment:
```suggestion
In case you lose access to your authenticator app, you can use the recovery codes to sign in to your account. To do that, click on the **recovery code or begin 2FA account recovery** link below the **Verify** button. This will redirect you to a similar view as the current one, but instead of code from the authenticator app, you will need to enter one of your 16 recovery codes which you have received during the setup process.
```